### DISCUSSION TITLE
Third song in every single week is missing, freeplay crashes the game.

### DISCUSSION THREAD

**Discussion by WarpedLGR** [FunkinCrew/Funkin#1524] [Compiling help] [answered]
Recently i've been compiling the game, so i could hopefully try and mod it, however when i opened it, all of the Third songs in all of the weeks (dadbattle,blammed,milf,winter horroland, thorns) were all gone, and when it reached these parts the game just crashed, the crashing also happens when i open the freeplay menu, how do i solve this?

**Comment by Trayfellow** [comment]
I also have this issue during story mode, the third/last songs are gone, but ik how to fix the freeplay problem, this videos description has a code that u can paste into the NGio.hx file and it should work

https://youtu.be/9dfkMgrjDOw

btw make sure to replace the entire ngio.hx script with the one in the description

**Comment by R-code-0** [comment]
Source not modified?

**Comment by oketariq** [comment]
Third song in every week missing is a Haxe/Flixel update bug. If you updated to 4.2.1 haxe and flixel 4.9.0 the issue will appear.

**Comment by Trayfellow** [comment, marked as answer]
> Third song in every week missing is a Haxe/Flixel update bug. If you updated to 4.2.1 haxe and flixel 4.9.0 the issue will appear.

im using haxe 4.1.5 rn, how do i go back to a version of flixel that fixes this issue?

edit: i found the command needed in the haxe documentation thing, I'll give an update once i finish trying it

edit 2 : i installed a different version of flixel, and still no luck :(

edit 3: omg i fixed it guys ok make sure the versions of your guys's libraries are like mine 
![image](https://user-images.githubusercontent.com/48515908/115024408-401b5100-9eea-11eb-86cc-64296b853a51.png)

**Comment by dariya-rogozinskaya** [comment]
> edit 3: omg i fixed it guys ok make sure the versions of your guys's libraries are like mine

It helped, thank you so much!
